CHANGELOG.md

```diff
@@ -0,0 +1,127 @@
+# Git Changelog plugin changelog
```

Comment:
Yes, it is nice, i would consider it as part of a release, but not maintained as source file.